unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDFactory.cs

```diff
@@ -34,7 +34,10 @@ public virtual IHUD CreateHUD(HUDElementID hudElementId)
                 hudElement = new MinimapHUDController(MinimapMetadataController.i, new WebInterfaceHomeLocationController(), DCL.Environment.i);
                 break;
             case HUDElementID.PROFILE_HUD:
-                hudElement = new ProfileHUDController(new UserProfileWebInterfaceBridge());
+                hudElement = new ProfileHUDController(new UserProfileWebInterfaceBridge(),
+                    new SocialAnalytics(
```

Comment:
it might be out of the scope of your task but let's think if we can rework it:
1. It seems redundant to instantiate a class that looks like a service and should exist in a single 
2. The same applies to `UserProfileWebInterfaceBridge`
3. The former makes sense if we split responsibility of `SocialAnalytics` in multiple classes

What do you think? Should we prepare a specification for refactoring?

Reply:
Yeah it makes sense :)